frontend/src/App.scss

```diff
@@ -66,23 +66,28 @@ div.bootstrap-table {
   word-break: break-word;
 }
 
+select.form-control.is-invalid {
```

Comment:
Prevent the error icon from displaying inline in the select which draws over the down arrow.